packages/react/src/components/TeachingBubble/TeachingBubble.types.ts

```diff
@@ -125,6 +125,12 @@ export interface ITeachingBubbleProps extends React.RefAttributes<HTMLDivElement
    * Defines the element id referencing the element containing the description for the TeachingBubble.
    */
   ariaDescribedBy?: string;
+
+  /**
+   * Sets the heading level used by the TeachingBubble headline, if headline is provided
```

Comment:
It would probably be good to clarify in the docs that this is for `aria-level` and doesn't actually render an `h3` or whatever. Or possibly just change the name to `ariaLevel`?